processor/k8sprocessor/options.go

```diff
@@ -68,31 +70,37 @@ func WithExtractMetadata(fields ...string) Option {
 	return func(p *kubernetesprocessor) error {
 		if len(fields) == 0 {
 			fields = []string{
-				metdataNamespace,
+				metadataCluster,
+				metadataContainerNames,
+				metadataDeployment,
+				metadataHostName,
```

Comment:
Can we reduce the scope of attributes added by default to the original list?
Container names seems to be not what everyone wants to be enabled by default, given that they are not particularly related to spans as must as pods are. Also hostName doesn't seems to be something that used broadly. Can we have them as optional?

Reply:
Agreed. Done.